web/drag.and.drop.upload.php

```diff
@@ -129,7 +131,8 @@ function readfiles(files) {
     // now post a new XHR request
     if (tests.formdata) {
       var xhr = new XMLHttpRequest();
-      xhr.open('POST', '/devnull.php');
+      // @robkooper says xhr.open fetches the file from the server side 
+      xhr.open('POST', '/dbfiles'); // changed destination directory
```

Comment:
Where exactly is /dbfiles relative to in your file structure -- does that imply a subfolder in the web folder or an absolute path relative to the root directory?